client/web/src/enterprise/insights/core/backend/utils/errors.ts

```diff
@@ -0,0 +1,6 @@
+export class InsightStillProcessingError extends Error {
```

Comment:
Would it make sense to use the same name for a class as the one used in the `name` property?

```suggestion
export class InsightInProcessError extends Error {
```